apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp

```diff
@@ -124,14 +124,15 @@ static void utc_media_MediaRecorder_setVolume_n(void)
 	mr.create();
 	mr.setDataSource(std::move(dataSource));
 	mr.prepare();
-	TC_ASSERT_EQ("utc_media_mediarecorder_setVolume", mr.setVolume(11), RECORDER_ERROR);
+	TC_ASSERT_EQ("utc_media_mediarecorder_setVolume", mr.setVolume(11), RECORDER_OK);
```

Comment:
This is a negative test case.
Is there any out of boundary value that returns RECOREDER_ERROR?

Reply:
Yes, the logic is changed that the bigger value than the maximum volume is changed into the maximum automatically inside our audio manager.

Reply:
we should add for negative case
for example)
mr.getVolume(&volume)
TC_ASSERT_EQ(&volume, 11);

Reply:
Hmm.. I understand what you are talking about.
Then I think it is more proper to delete line 124 to 127.
How do you think about it?

Reply:
Well.. @junmin-kim , the negative test case is in line 123.